README.md

```diff
@@ -98,3 +98,307 @@ The changelog can be found [here](https://github.com/rust-secure-code/cargo-geig
 [cargo]: https://crates.io/crates/cargo
 [cargo-geiger]: https://crates.io/crates/cargo-geiger
 [changelog]: https://github.com/rust-secure-code/cargo-geiger/blob/master/CHANGELOG.md
+
+## Cargo Geiger Safety Report
+```
+
+Metric output format: x/y
+    x = unsafe code used by the build
+    y = total unsafe code found in the crate
+
+Symbols: 
+    :) = No `unsafe` usage found, declares #![forbid(unsafe_code)]
+    ?  = No `unsafe` usage found, missing #![forbid(unsafe_code)]
+    !  = `unsafe` usage found
+
+Functions  Expressions  Impls  Traits  Methods  Dependency
+
+0/0        0/0          0/0    0/0     0/0      :) cargo-geiger 0.10.2
+10/10      210/210      0/0    0/0     1/1      !  ├── anyhow 1.0.33
+4/4        341/347      0/0    0/0     3/3      !  ├── cargo 0.47.0
+10/10      210/210      0/0    0/0     1/1      !  │   ├── anyhow 1.0.33
+2/2        45/45        0/0    0/0     0/0      !  │   ├── atty 0.2.14
+0/5        12/259       0/0    0/0     2/22     !  │   │   └── libc 0.2.79
+0/0        0/0          0/0    0/0     0/0      ?  │   ├── bytesize 1.0.1
+0/0        0/0          0/0    0/0     0/0      ?  │   │   └── serde 1.0.117
```

Comment:
Random idea: Would inline html be desirable when targeting GitHub markdown? We could merge this PR now and discuss it in a different issue.

Reply:
I think this would definitely be worth doing, possibly not only for GitHub Markdown, but if people wanted to use the reports elsewhere? I can land another PR after this one to implement the inline HTML?

Reply:
>  I can land another PR after this one to implement the inline HTML?

Sounds good!